Tests/SourceKittenFrameworkTests/SwiftDocsTests.swift

```diff
@@ -2,19 +2,32 @@ import Foundation
 @testable import SourceKittenFramework
 import XCTest
 
+extension FileManager {
+    #if os(Windows)
+    static let pathSeparator = "\\"
+    #else
+    static let pathSeparator = "/"
+    #endif
+}
+
 func compareJSONString(withFixtureNamed name: String,
                        jsonString: CustomStringConvertible,
                        rootDirectory: String = fixturesDirectory,
                        file: StaticString = #file,
                        line: UInt = #line) {
+    let jsonString = String(describing: jsonString)
+
     // Strip out fixtures directory since it's dependent on the test machine's setup
-    let escapedFixturesDirectory = rootDirectory.replacingOccurrences(of: "/", with: "\\/")
-    let jsonString = String(describing: jsonString).replacingOccurrences(of: escapedFixturesDirectory, with: "")
+    let escapedFixturesDirectory = URL(fileURLWithPath: rootDirectory).standardizedFileURL.withUnsafeFileSystemRepresentation {
+        String(cString: $0!) + FileManager.pathSeparator
+    }.replacingOccurrences(of: "\\", with: "\\\\").replacingOccurrences(of: "/", with: "\\/")
```

Comment:
Same here, could you please use raw string literals to avoid having to escape `\` which adds a lot of noise?